packages/design-system-web/src/components/geov-map-places/geov-map-places.tsx

```diff
@@ -0,0 +1,293 @@
+            //   * --circle-color-large, 40px circles when point count is greater than or equal to 750
+            'circle-color': ['step', ['get', 'point_count'], customColors[0], 100, customColors[1], 750, customColors[2]],
+            'circle-radius': ['step', ['get', 'point_count'], 20, 100, 30, 750, 40],
+          },
+        });
+
+        map.addLayer({
+          id: 'cluster-count',
+          type: 'symbol',
+          source: 'places',
+          filter: ['has', 'point_count'],
+          layout: {
+            'text-field': '{point_count_abbreviated}',
+            'text-size': 12,
+          },
+        });
+
+        //   Add popups to the markers
+        const handleMarkerClick = e => {
+          const coordinates = e.features[0].geometry.coordinates;
+          const description = e.features[0].properties.name;
+          const link = this.projectID ? `${e.features[0].properties.link}?p=${this.projectID}` : e.features[0].properties.link;
+          new Popup().setLngLat(coordinates).setHTML(`<div><a href = "${link}" target="_blank">${description}</a></div>`).addTo(map);
+        };
+
+        // An on click event listener for the "unclustered-point" layer
+        map.on('click', 'unclustered-point', handleMarkerClick);
+
+        // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=700533
+        // From https://webcompat.com/issues/64352
+        function ensureSVGWidthHeight(svgElem) {
+          if (!svgElem.getAttribute('width') || !svgElem.getAttribute('height')) {
+            const viewBox = svgElem.getAttribute('viewBox');
+            if (viewBox) {
+              const match = viewBox.match(/\w+ \w+ (\w+) (\w+)/);
+              const viewBoxWidth = match[1];
+              const viewBoxHeight = match[2];
+              svgElem.setAttribute('width', viewBoxWidth + 'px');
+              svgElem.setAttribute('height', viewBoxHeight + 'px');
+            }
+          }
+          return svgElem;
+        }
+
+        function svgStringToImageSrc(svgString) {
+          svgString = svgString.substring(svgString.indexOf('<svg'));
+          const parser = new DOMParser();
+          const svgXML = parser.parseFromString(svgString, 'image/svg+xml');
+          const fixedSvgXML = ensureSVGWidthHeight(svgXML.rootElement);
+          const fixedSvgString = new XMLSerializer().serializeToString(fixedSvgXML);
+          return 'data:image/svg+xml;charset=utf-8,' + encodeURIComponent(fixedSvgString);
+        }
+
+        const image = new Image(20, 20);
+        image.onload = () => {
+          map.addImage('flag', image, { pixelRatio: window.devicePixelRatio });
+          map.addLayer({
+            id: 'unclustered-point',
+            type: 'symbol',
+            source: 'places',
+            filter: ['!', ['has', 'point_count']],
+            layout: {
+              'icon-image': 'flag',
+              'icon-size': 1,
+              'icon-allow-overlap': true,
+            },
+          });
+        };
```

Comment:
Clever solution!
Why do you prefer a symbol over the circle?
IMO the flag looks a bit strange. If it is a circle without number inside, it will be visually distinguishable from clustered points.
Otherwise you could also use the ionicon "ellipse".

Reply:
Because of distinguishability, was my idea, but implementing simple circles saves us a ton of workarounds - so of course I'll do that! :-)

Reply:
Do you want me to include a fourth color or do I reuse one of the circle colors? or do I set it to black? @joschne

Reply:
I'd prefer a forth color @flicksolutions .